cw_bitcoin/lib/electrum_wallet_addresses.dart

```diff
@@ -317,12 +319,79 @@ abstract class ElectrumWalletAddressesBase extends WalletAddresses with Store {
     return address;
   }
 
-  String getAddress(
-          {required int index,
-          required Bip32Slip10Secp256k1 hd,
-          BitcoinAddressType? addressType}) =>
+  String getAddress({
+    required int index,
+    required Bip32Slip10Secp256k1 hd,
+    BitcoinAddressType? addressType,
+  }) =>
       '';
 
+  Future<String> getAddressAsync({
+    required int index,
+    required Bip32Slip10Secp256k1 hd,
+    BitcoinAddressType? addressType,
+  }) async =>
+      getAddress(index: index, hd: hd, addressType: addressType);
+
+  void addBitcoinAddressTypes() {
+    final lastP2wpkh = _addresses
+        .where((addressRecord) =>
+            _isUnusedReceiveAddressByType(addressRecord, SegwitAddresType.p2wpkh))
+        .toList()
+        .last;
```

Comment:
I'm not sure why we pick the last address, but maybe it has something to do with preventing address re-use? I made a separate addLitecoinAddressTypes() function for now but will hold off on getting the first instead of last until I'm sure it won't cause unintended address re-use